src/main/java/cz/it4i/parallel/ui/LastFormLoader.java

```diff
@@ -0,0 +1,76 @@
+package cz.it4i.parallel.ui;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.util.Base64;
+
+import org.scijava.plugin.Parameter;
+import org.scijava.prefs.PrefService;
+
+import cz.it4i.parallel.runners.ImageJServerRunnerSettings;
+import lombok.extern.slf4j.Slf4j;
+
+//Code responsible for loading and storing the last used form values:
+@Slf4j
+public class LastFormLoader {
+	
+	private String formStorage;
+	
+	@Parameter
```

Comment:
@Parameter is not required - it is set in constructor.

Reply:
Removed the @Parameter.